lib/manageiq/appliance_console/prompts.rb

```diff
@@ -212,7 +212,6 @@ def ask_with_menu(prompt, options, default = nil, clear_screen_after = true)
 
     def just_ask(prompt, default = nil, validate = nil, error_text = nil, klass = nil)
       ask("Enter the #{prompt}: ", klass) do |q|
-        q.readline = true
```

Comment:
Should we set it explicitly to false so we can easily update it "true" after we fix it?

Reply:
🤷🏻‍♂️

Reply:
I'm fine either way.